services/web-app/src/functions/add-user/index.ts

```diff
@@ -1,11 +1,30 @@
 import { DynamoDBStreamEvent } from "aws-lambda";
 import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
 import { PutCommand, DynamoDBDocumentClient } from "@aws-sdk/lib-dynamodb";
-import { buildResourceName } from '../../../../core/helpers';
+import fetch from "node-fetch";
+import { getVariableFromSSM } from "../../../../core/functions/helpers/getVariable";
 
 const client = new DynamoDBClient({});
 const docClient = DynamoDBDocumentClient.from(client);
 
+const postEmail = async(email: string, name: string) => {
+  return await fetch(process.env.CLOUDFLARE_WORKER_URL ?? "", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+      "Authorization": await getVariableFromSSM(
+        process.env.CLOUDFLARE_WORKER_TOKEN_NAME ?? ""
+      ),
+    },
+    body: JSON.stringify({
+      "to": {"email": email, "name": name },
+      "from": { "email": "test@oriontools.ai", "name": "Orion Tools" },
```

Comment:
better to have it come from noreply@oriontools.ai. Name: 'Matt from Orion Tools'

Reply:
can you change this